common/api/summary/imodel-bridge.exports.csv

```diff
@@ -3,5 +3,10 @@ Release Tag;API Item
 beta;BridgeJobDefArgs
 public;BridgeLoggerCategory
 beta;BridgeRunner
+beta;ChangeResults
```

Comment:
Mind deleting this file and then re-running extract-api? It should re-generate as `itwin-connector-framework.exports.csv`.